common/persistence/class.PhpFileDriver.php

```diff
@@ -260,14 +260,14 @@ private function writeFile($id, $value, $preWriteValueProcessor = null)
 
     /**
      * Create directory and suppress warning message
-     * @param $path
+     * @param string $path
      * @param int $mode
      * @return bool
      */
     private function makeDirectory(string $path, int $mode)
     {
-        if (is_dir($path) || @mkdir($path, $mode, true)) {
-            return true;
+        if (!is_dir($path)) {
+            return @mkdir($path, $mode, true);
```

Comment:
Is it good, if mkdir is not successful (=returns false), then the makeDirectory method just returns false without logging anything?

Reply:
Good point, changed it a bit